src/libs/ReportUtils.ts

```diff
@@ -4316,6 +4316,21 @@ function navigateToPrivateNotes(report: Report, session: Session) {
     Navigation.navigate(ROUTES.PRIVATE_NOTES_LIST.getRoute(report.reportID));
 }
 
+function getReportFieldTitle(report: OnyxEntry<Report>, reportField: PolicyReportField) {
```

Comment:
Can we include a comment to explain the purpose and the expected output?

Reply:
Added.

Reply:
Thanks! Can we also add the return type?

Reply:
I think the return type will be only string, right?

Reply:
Yep. Added.